ci/docker/runtime_functions.sh

```diff
@@ -371,7 +371,6 @@ unittest_ubuntu_python2_cpu() {
     export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
     nosetests-2.7 --verbose tests/python/unittest
     nosetests-2.7 --verbose tests/python/train
-    nosetests-2.7 --verbose tests/python/quantization
```

Comment:
Please leave this in. It's supposed to be here, right @reminisce ?

Reply:
@marcoabreu We cannot pass the `test_quantization_mkldnn.py` when building without mkldnn. So I skip this line or we can put the test file in a separate dictionary?

Reply:
Please create a dir called tests/python/mkl and put MKL specific tests there.

Reply:
ok:)

Reply:
@jinhuang415 @xinyu-intel could you please put the line back?

Reply:
@zheng-da We cannot pass this test_quantization.py when building without mkldnn.